manifests/install.pp

```diff
@@ -33,6 +36,7 @@
   $target_dir   = undef,
   $with_npm     = true,
   $make_install = true,
+  $python_package = 'python',p
```

Comment:
I assume that the ``p`` is just a typo.
Please check that every ``=`` is aligned properly:

``` puppet
$target_dir     = undef,
$with_npm       = true,
$make_instal    = true,
$python_package = true,
```